### PR TITLE
*: Prepare v0.45.1

### PR DESCRIPTION
**Pull request by mxinden**
# Description

Proposing to release `v0.45.1`. Anything else you would like to see included?

To be merged before the release:

- https://github.com/libp2p/rust-libp2p/pull/2699

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,15 @@
...
 # `libp2p` facade crate
 
+# 0.45.1
+
+- Update individual crates.
+    - Update to [`libp2p-dcutr` `v0.3.1`](protocols/dcutr/CHANGELOG.md).
+    - Update to [`libp2p-identify` `v0.36.1`](protocols/identify/CHANGELOG.md).
+    - Update to [`libp2p-kad` `v0.37.1`](protocols/kad/CHANGELOG.md).
+    - Update to [`libp2p-relay` `v0.9.1`](protocols/relay/CHANGELOG.md).
+    - Update to [`libp2p-swarm` `v0.36.1`](swarm/CHANGELOG.md).
```
Comment (thomaseizinger) [inline CHANGELOG.md:53]: Previous entries don't have whitespace at the front.

```suggestion
- Update to [`libp2p-dcutr` `v0.3.1`](protocols/dcutr/CHANGELOG.md).
- Update to [`libp2p-identify` `v0.36.1`](protocols/identify/CHANGELOG.md).
- Update to [`libp2p-kad` `v0.37.1`](protocols/kad/CHANGELOG.md).
- Update to [`libp2p-relay` `v0.9.1`](protocols/relay/CHANGELOG.md).
- Update to [`libp2p-swarm` `v0.36.1`](swarm/CHANGELOG.md).
```

**Review by thomaseizinger — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mxinden) [reply CHANGELOG.md:53]: Not sure I am following. Isn't this the same for v0.45.0 below?

Comment (thomaseizinger) [reply CHANGELOG.md:53]: The diff view doesn't show the whitespace 🙃

**Review by elenaf9 — APPROVED** [review]

**Comment by mxinden** [comment]
Published and tagged the crates below:

- libp2p-dcutr-0.3.1
- libp2p-identify-0.36.1
- libp2p-kad-0.37.1
- libp2p-relay-0.9.1
- libp2p-swarm-0.36.1
- v0.45.1

**Pull request merged**